z/Zeno.zno

```diff
@@ -0,0 +1,3 @@
+program print_hello
+  put "Hello, world!"
```

Comment:
```suggestion
  put "Hello World"
```

Reply:
I have changed ` put "Hello, world!"` to ` put "Hello World"`